components/apimgt/org.wso2.carbon.apimgt.api/src/main/java/org/wso2/carbon/apimgt/api/APIProvider.java

```diff
@@ -954,6 +1007,17 @@ boolean updateAPIforStateChange(APIIdentifier identifier, APIStatus newStatus,
      */
     boolean isCertificatePresent(int tenantId, String alias) throws APIManagementException;
 
+    /**
+     * Method to check whether an client certificate for the given alis is present in trust store and whether it can
+     * be modified by current user.
+     *
+     * @param tenantId Id of the tenant.
```

Comment:
missing colon

Reply:
AFAIK colon is not a mandatory in java doc comment. However since we have followed the same pattern in this interface in the some other methods, I will make changes to include the colon only in this particular interface.